.github/workflows/main.yml

```
@@ -96,6 +96,10 @@ jobs:
             rust: stable
             target: x86_64-unknown-linux-musl
             bin: cargo-machete
+          - os: ubuntu-22.04-arm
```

Runner images have been updated:
```suggestion
          - os: ubuntu-24.04-arm
```